share/picongpu/examples/FoilLCT/include/picongpu/param/memory.param

```diff
@@ -49,6 +49,7 @@ namespace picongpu
      * volume of a superCell must be <= 1024
      */
     using SuperCellSize = mCT::Int<16, 16>;
+    static constexpr uint32_t numFrameSlots = pmacc::math::CT::volume<SuperCellSize>::type::value;
```

Comment:
the rest of `memeory.param` has a comment for this new line of code. This one doesn't.

Reply:
fixed